routing/route.hpp

```diff
@@ -135,6 +147,10 @@ class Route
   inline void SetStreetNames(TStreets && v) { m_streets = move(v); }
   inline void SetAltitudes(feature::TAltitudes && v) { m_altitudes = move(v); }
   inline void SetTraffic(vector<traffic::SpeedGroup> && v) { m_traffic = move(v); }
+  inline void SetSegmentInfo(vector<SegmentInfo> && v)
+  {
+    m_segmentInfo = std::forward<vector<SegmentInfo>>(v);
```

Comment:
Don't forget template argument, otherwise ```&&``` is just an rvalue-reference, not a universal reference.
```c++
template<typename SI>
void SegSegmentInfo(SI && v)
{
  m_segmentInfo = std::forward<SI>(v);
}
```